DiffPlex.Wpf/Controls/InternalLinesViewer.xaml.cs

```diff
@@ -106,10 +252,13 @@ public StackPanel Add(int? number, string operation, string value, string change
 
         panel.Children.Add(text);
         ValuePanel.Children.Add(panel);
+        ValuePanel.CanHorizontallyScroll = true;
```

Comment:
Why do we need to set explicitly, I saw earlier you were creating dependency properties, shouldn't these also react to that?

Reply:
We do not. You're correct. I've removed this